### ISSUE TITLE
Amount Awarded (GBP) filters non-GBP currencies 

### ISSUE THREAD

**Issue by robredpath** [ThreeSixtyGiving/grantnav#791]
**Describe the bug**

When the `Amount Awarded (GBP)` filter is used, the results set still includes non-GBP grants in the same numeric range

**To Reproduce**
1. Do a blank search
2. Click £5,000-£10,000 in the Amount Awarded (GBP) facet

**Expected behavior**
No USD grants would be in the results set

**Screenshots**

![Screenshot 2021-10-05 at 14 24 53](https://user-images.githubusercontent.com/641009/136031741-5e54966f-1cc6-4a77-8a74-770a6e40c6ee.png)

**Desktop (please complete the following information):**
macOS 11.6 / Firefox 92.1

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by mariongalley** [comment]
@michaelwood What do you think about this? I feel like having Amount Awarded filters work with not GBP currencies is a good thing (although it doesn't necessarily make sense to filter across multiple currencies)

**Comment by michaelwood** [comment]
Yeah, my thought was that maybe for each currency that is selected there is an amount awarded filter box. 

![image](https://github.com/ThreeSixtyGiving/grantnav/assets/493697/4e660596-5c35-476c-ad65-35405b0cb09f)

This is a fairly big piece of work because none of the current tooling is "currency aware", e.g. for the current filter we're simply just counting/filtering on the number of instances of the ISO 4217, string (GBP, USD etc)

**Comment by mariongalley** [comment]
I guess that is the most logical way to handle it, but seems excessive in terms of the amount of space taken up. I'm kind of happy with the current functionality tbh - most users will ignore the currency picker unless they're actively looking for non-GBP grants, so giving them 7+ amount filters when they're trying to find grants relevant to a theme or a place seems unnecessary.

We could hide the amount filter and say "You must choose 1 currency to filter amounts", but that seems to introduce an unnecessary click when most grants are GBP and most users are interested in GBP. Having it filter all currencies seems like an acceptable solution for this use-case (even if the non GBP grants are less relevant as not being filtered accurately)
